api/swagger.yaml

```diff
@@ -4663,7 +4663,7 @@ paths:
               AppArmorProfile:
                 type: "string"
               ExecIDs:
-                type: "string"
+                type: "array"
```

Comment:
hm, just realised this also needs a type for items in the array. Opened a follow-up PR https://github.com/moby/moby/pull/37121

Reply:
I think there are also some other issues with the swagger, there's some strings which seem to wrap on to new lines, breaking the JSON.

I can look into those and make another PR when I have some time.

Reply:
strings breaking on new lines should be ok with the right delimiter (but won't work with just quotes indeed)